ftw/mobile/scss/mobile-menu-buttons.scss

```diff
@@ -152,20 +152,22 @@ $color-mobile-current: $color-gray-dark !default;
         > a {
           padding-left: $padding-horizontal * 2;
           background-color: $color-gray-light;
+          width: calc(100% - 70px - #{$size-mobile-button});
           &.mobileActionNav {
             padding-left: $padding-horizontal;
           }
         }
       }
       border-bottom: 1px solid $color-gray-dark;
+      width: calc(100% - 24px);
     }
   }
 
   .navCurrentNode > a {
     background-color: $color-mobile-current;
     @include auto-link-color($color-mobile-current);
     font-weight: bold;
-    width: 100%;
+    width: calc(100% - 47px);
```

Comment:
Where do these numbers come from? @bierik always tells me to not use or at least comment on arbitrary numbers in CSS.

Reply:
I removed the use of arbitrary numbers. I work with 100% and the already defined values only now.